xml/obs_best_practice_webui_usage.xml

```diff
@@ -106,6 +106,8 @@
     On the "Subprojects" tab you can find a list subprojects that belong to a
     project. To create a new subproject click on the "New subproject" link, fill
     in the form and press the "Create project" button.
+    Please note that maintainer of upper projects always can modify the sub projects.
```

Comment:
Just a suggestion. This could be moved into a note element. But then I would remove the please note part;-)